### PR TITLE
Extract out xpath to a plugin.

### PR DESCRIPTION
**Pull request by andrasio**

### CODE REVIEW

File: wix/main.wxs [hunk 1]
```diff
@@ -232,6 +232,14 @@
                                 Source='target\$(var.Profile)\nu_plugin_extra_chart_line.exe'
                                 KeyPath='yes'/>
                         </Component>
+                        <Component Id='binary18' Guid='*' Win64='$(var.Win64)'>
+                            <File
+                                Id='exe17'
```
Comment (fdncred) [inline wix/main.wxs:237]: this id should be `exe18`

also, down below, in the `ComponentRef` section there needs to be `binary18`

Comment (andrasio) [reply wix/main.wxs:237]: Good stuff. Will change soon.

File: crates/nu_plugin_xpath/src/xpath.rs [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+    let context = Context::new();
+
+    // leaving this here for augmentation at some point
+    // build_variables(&arguments, &mut context);
+    // build_namespaces(&arguments, &mut context);
+
+    let res = xpath.evaluate(&context, document.root());
+
+    // Some xpath statements can be long, so let's truncate it with ellipsis
+    let mut key = query_string.clone();
```
Comment (fdncred) [inline crates/nu_plugin_xpath/src/xpath.rs:64]: i'm fine with leaving this but we were talking about this in discord today, should the xpath query, which is truncated to 20 chars, be the column label, or should we have a default label of `result` or `value` or something like that?

Comment (andrasio) [reply crates/nu_plugin_xpath/src/xpath.rs:64]: Thought of it but decided to leave it as it was. Up to you!

**Pull request merged**